data/reporter.htm

```diff
@@ -22,6 +22,7 @@
     </div>
 
     <div id="buttons">
+        <span id="appe10senabled"></span>
```

Comment:
I'd sugget to use camelCase or whatever-the-dashes-case-is-called here for consistency